packages/fiori-elements-writer/test/lrop.test.ts

```diff
@@ -29,6 +29,18 @@ describe(`Fiori Elements template: ${TEST_NAME}`, () => {
                 service: v4Service
             } as FioriElementsApp<LROPSettings>
         },
+        {
+            name: 'lropV4noUi5Version',
+            config: {
+                ...Object.assign(feBaseConfig('felropui5', false), {
```

Comment:
Isn't that redundant and also potentially wrong due to the assignment side effect ?

An object spread will already create a new object so you don't need to also call object.assign, except if you're really looking for the side effect of updating the feBaseConfig with your new value (like you're doing there).

If this was really intended i would still encourage to rewrite this part as it is very difficult to understand all that is happening here :)

Reply:
Argh ... I haven't looked at this one, just copied an existing test case and changed the input for the new case to test. Do you mind if I leave it in, and touch it separately?

Reply:
sure, it's in the test code so it's not too bad but i do hope it doesn't hide a real issue :)

Reply:
I don't think so, the code with or without the `Object.assign` would in this case result in the same.